openverse-api/catalog/api/models/models.py

```diff
@@ -14,6 +15,10 @@ class ContentProvider(models.Model):
         verbose_name='Hide content'
     )
     notes = models.TextField(null=True)
+    media_type = models.CharField(
```

Comment:
I think we should add a node somewhere where we create the content providers about the different internal representation for providers that have several content types (wikimedia_image, wikimedia_audio)

Reply:
Good suggestion @obulat

Reply:
Added usage comment in 99c18e0.

Reply:
Added usage comment in 99c18e0.